README.md

```diff
@@ -434,6 +508,10 @@ func (dest *AdvancedFetchRequest) UnmarshalJSON(buf []byte) error {
 Pros of this approach:
 
 - No dependency.
+- You get to write more Go code!
+- You get to spend more time debugging Go code!
+- You get to spend more time debugging Go code *in production*!
+- You get to write more tests!
```

Comment:
I do appreciate the irony, but I'm not sure other team members will :sweat_smile:

Just bringing this to your attention, your call.

Reply:
I _think_ that other team members will be ok with it.

Famous last words :)